cmd/protogen/disable_utf8.go

```diff
@@ -92,8 +104,10 @@ func (v *disableUtf8Validation) visit(n ast.Node) bool {
 			newElts := make([]ast.Expr, len(newArr))
 			for i, v := range newArr {
 				newElts[i] = &ast.BasicLit{
-					Kind:  token.INT,
-					Value: fmt.Sprintf("%#02x", v),
+					// steal positions from original list to preserve newlines
```

Comment:
I read through a bunch of the formatter/printer.. it looks like adding newlines in exactly the right places (and specifically adding more than there were before) is quite a lot more work. this is a pretty easy way to just get _some_ newlines in there.